packit/constants.py

```diff
@@ -212,16 +212,9 @@ def url(self) -> str:
 )
 SYNC_RELEASE_PR_INSTRUCTIONS = (
     "\n---\n\n"
-    "If you need to do any change in this pull request, you need to locally fetch the "
-    "source branch of it and push it (with a fix) to your fork "
-    "(as it is not possible to push to the branch created in the Packit's fork):\n"
-    "```\ngit fetch https://src.fedoraproject.org/forks/{user}/rpms/{package}.git "
-    "refs/heads/*:refs/remotes/{user}/*\n"
-    "git checkout {user}/{branch}\n```\n"
```

Comment:
we could still provide the `git checkout` command as well...and maybe would it make sense to alternatively recommend adding the remote and fetching it instead of the clone if they have the repo cloned already?

Reply:
Oh, right, I didn't realize that. Ok, it makes sense to leave the other way there as an alternative.